include/configs/hmi.h

```diff
@@ -0,0 +1,154 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * (C) Copyright 2012-2016 Stephen Warren
+ */
+
+#ifndef __CONFIG_H
+#define __CONFIG_H
+
+#include <linux/sizes.h>
+#include <asm/arch/timer.h>
+#include <linux/stringify.h>
+
+#ifndef __ASSEMBLY__
+#include <asm/arch/base.h>
+#endif // __ASSEMBLY__
+
+#define CONFIG_SYS_TIMER_RATE		1000000
```

Comment:
Apparently, the base was not the rpi.h file https://github.com/fancom/u-boot/blob/fancom/master/include/configs/rpi.h, but why not? That file differs from what we have here...

Reply:
Fixed.